File: Tema 03 - Web API/PetShelter/PetShelter.DataAccessLayer/Models/Fundraiser.cs
namespace PetShelter.DataAccessLayer.Models
{
    public class Fundraiser : IEntity
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal DonationTarget { get; set; }
        public Person Owner { get; set; }
        public int? OwnerId { get; set; }
        public ICollection<Person> Donors { get; set; }
        public ICollection<Donation> Donations { get; set; }
        public FundraiserStatus Status { get; set; }
        public DateTime CreationTime { get; set; }

        public decimal RaisedAmount
        {
            get; set;
        }

        public DateTime DueDate { get; set; }
    }

}

File: Tema 05 - Typescript/after/PetShelterFrontend/src/models/Fundraiser.ts
import type { Person } from "./Person";
import type { FundraiserStatus } from "./FundraiserStatus";

export class Fundraiser{
     title:string;
     description : string;
     owner : Person;
     dueDate : Date;
     status: FundraiserStatus;
     creationTime: Date;
     raisedAmount : Number;

     constructor(title:string, description:string,owner:Person, dueDate:Date,status:FundraiserStatus,creationTime: Date,raisedAmount:Number){
        this.title=title;
        this.description=description;
        this.owner=owner;
        this.dueDate=dueDate;
        this.status=status;
        this.creationTime=creationTime;
        this.raisedAmount=raisedAmount;
     }


}